ch_6_Multi-Language_Support/6_1_ChatGLM2-6B.ipynb

```diff
@@ -167,7 +167,13 @@
    "source": [
     "### 6.1.4.3 Stream Chat\n",
     "\n",
-    "ChatGLM2-6B support streaming output function `stream_chat`, which enable the model to provide a streaming response word by word. However, other models may not provide similar APIs, if you want to implement general streaming output function, please refer to [Chapter 4.1](../ch_4_Run_Models/4_1_Run_Transformer_Models.ipynb)."
+    "ChatGLM2-6B support streaming output function `stream_chat`, which enable the model to provide a streaming response word by word. However, other models may not provide similar APIs, if you want to implement general streaming output function, please refer to [Chapter 4.1](../ch_4_Run_Models/4_1_Run_Transformer_Models.ipynb).\n",
+    "\n",
+    "> **Note**\n",
+    ">\n",
+    "> To successfully observe the text streaming behavior in standard output, we need to set the environment variable `PYTHONUNBUFFERED=1 `to ensure that the standard output streams are directly sent to the terminal without being buffered first.\n",
+    ">\n",
+    "> The [Hugging Face *transformers* streamer classes](https://huggingface.co/docs/transformers/main/generation_strategies#streaming) is currently being developed and is subject to future changes.\n"
```

Comment:
Please remove this sentence :) "The [Hugging Face *transformers* streamer classes](https://huggingface.co/docs/transformers/main/generation_strategies#streaming) is currently being developed and is subject to future changes.\n""
ChatGLM is not using streamer for stream chat

Reply:
Sure.